lib/improver/tests/lapse_rate/Test_LapseRate.py

```diff
@@ -290,6 +344,85 @@ def test_lapse_rate_limits(self):
                                                     self.land_sea_mask)
         self.assertArrayAlmostEqual(result.data, expected_out)
 
+    def test_specified_max_lapse_rate(self):
```

Comment:
Is this test just checking the code runs (and gets output) if you use a max lapse rate outside of the constraint? As I understand the ticket (which might not be right) it wants you to test non default values. However these are values which shouldn't be used (considering the constrains in lapse_rate.py?

Reply:
The code is checking that the code is correctly applying a _user specified_ max lapse rate constraint to the calculated output, by allowing higher lapse rates. 

You can see in the previous test the default (no argument) max of  `-3*DALR` is used and applies a limit of `0.0294 K m-1` whereas this test with a higher max lapse rate of -`4*DALR` allows values up to `0.0392 K m-1`

See comments above regarding the constraints in`lapse_rate.py` only being default values and that using other constraint values is part of the plugins functionality.